build-and-test.ps1

```diff
@@ -23,9 +23,9 @@ $manifestRepo = $manifest.Repos[0]
 $activeOS = docker version -f "{{ .Server.Os }}"
 $builtTags = @()
 
-$buildFilter = $Filter
+$buildFilter = $VersionFilter
 if ($activeOS -eq "windows" -and $buildFilter -like "2.*") {
```

Comment:
This needs to be refactored some more.  The buildFilter should always be set to `$versionFilter/*/$OS*`.  If $versionFilter or $OS are not specified, their usage in the buildFilter should be `*`